### PR TITLE
fix: seedlot columns and area of use

### PR DESCRIPTION
**Pull request by RMCampos**
# Description
Closes no issue

#### New
- 2 SQL migrations: V21 and V22;
- Seedlot Seed Plan Zone entity class;

#### Changed
- Seedlot entity class

#### Removed
- Seedlot columns `tested_parent_tree_cont_pct` and `coancestry`;

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-685-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-685-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-685-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

**Review by craigyu — APPROVED** [review]
LGTM

File: backend/src/main/resources/db/migration/V22__area_of_use.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	add column latitude_min_max 			smallint,
+	add column latitude_sec_max 			smallint,
+	add column longitude_deg_min 			smallint,
+	add column longitude_min_min 			smallint,
+	add column longitude_sec_min 			smallint,
+	add column longitude_deg_max 			smallint,
+	add column longitude_min_max 			smallint,
+	add column longitude_sec_max 			smallint,
+	add column smp_mean_bv_growth			decimal(4, 1),
+	add column area_of_use_comment			varchar(2000);
```
Comment (mgaseta) [inline backend/src/main/resources/db/migration/V22__area_of_use.sql:41]: Maybe it's git fault but the spacing seems odd here, shouldn't it be in the same pattern as `V21__pt_number_pt_tested_pct.sql`.

File: backend/src/main/java/ca/bc/gov/backendstartapi/entity/seedlot/Seedlot.java [hunk 2]
```diff
@@ -119,6 +119,9 @@ public class Seedlot implements Serializable {
   @Column(name = "interm_facility_code", length = 3)
   private String interimStorageFacilityCode;
 
+  @Column(name = "interm_strg_locn", length = 55)
+  private String intermStorageLocationDescription;
```
[the remark this answers is not in the document]
Comment (RMCampos) [reply backend/src/main/java/ca/bc/gov/backendstartapi/entity/seedlot/Seedlot.java:123]: I created this variable name based on the column name. Nice catch!

File: backend/src/main/resources/db/migration/V22__area_of_use.sql (hunk 1 above)
Comment (RMCampos) [reply backend/src/main/resources/db/migration/V22__area_of_use.sql:41]: Yep. Forgot to replace tabs by space. Thanks!

Comment (RMCampos) [reply backend/src/main/resources/db/migration/V22__area_of_use.sql]: Done here: https://github.com/bcgov/nr-spar/pull/685/commits/48c103cd0870ae17ed4adadfc9b5cbd60e9e61c4

Comment (mgaseta) [reply backend/src/main/resources/db/migration/V22__area_of_use.sql]: Looking good!

**Pull request merged**